Dockerfile

```
@@ -28,7 +28,7 @@ RUN apt-get update \
         wget \
         unzip \
         zip \
-        gnupg
+        gnupg \
```

looks like a mistake?
```suggestion
        gnupg
```

---

Yes, typo. Now removed.